tensorflow/python/keras/layers/wrappers.py

```diff
@@ -361,6 +361,10 @@ class Bidirectional(Wrapper):
   Call arguments:
     The call arguments for this layer are the same as those of the wrapped RNN
       layer.
+    Beware that when passing the `initial_state` argument during the call of this
+    layer, the first N elements in the `initial_state` list will be passed to the
```

Comment:
I think the N is the major word here and doesn't carry much meaning. How about first half and second half?